setup-linux-bridge.sh

```diff
@@ -54,11 +47,13 @@ if [[ $(nmcli conn | grep -c brcnv) -eq 0 ]]; then
       nmcli conn mod "$secondary_profile_name" connection.autoconnect yes
       nmcli c delete $(nmcli c show |grep ovs-cnv |awk '{print $1}') || true
   else
+      ip link set bond0 down
```

Comment:
By configuring this using ip link, you are risking that NetworkManager will revert any changes you made. Isn't there a NetworkManager option to set what you do below? If not, we should open an RFE on NM.

Reply:
Looks like we need to change the command to: nmcli conn down/up bond0
I will try to ask the CU if he can do a recheck to make sure we are aligned and update

Reply:
Right. As long as the `tlb_dynamic_lb=0` attribute on line 35 works well, these three lines are redundant and should be dropped.

Reply:
Based on discussion with dcritch, we tried nmcli conn down ; echo into tlb_dynamic_lb; nmcli conn up, and the connection completely disappeared, no sysfs node available, so we switched to ip link calls, which worked. The long and short of it is that updates to tlb_dynamic_lb can't be done with the bond up.

Reply:
It's entirely possible NM isn't aware that tlb_dynamic_lb is actually valid for balance-alb in addition to balance-tlb.

Reply:
> That's right. `tlb_dynamic_lb=0` has no effect in NetworkManager (bug? seems that it should work if using `balance-tlb` according to NM QE tests I found[0].) Doing a `nmcli conn down/up` was too heavy and removed `/sys/class/net/bonding/bond0` altogether, while `ip link up/down` is lighter and keeps the structure under there so we can do the `echo`.

If it is not working with NetworkManager, please file a bug. It would also be good to add a comment to this script that this is a workaround for the bug and reference that bug to make this more clear.

Reply:
https://bugzilla.redhat.com/show_bug.cgi?id=1959934